extensions/spring-xd-extension-sqoop/src/main/java/org/springframework/xd/sqoop/SqoopTasklet.java

```diff
@@ -74,6 +86,9 @@ protected boolean isStoppable() {
 				}
 			}
 		}
+		for (Map.Entry e : hadoopProperties.entrySet()) {
+			command.add(SPRING_HADOOP_CONFIG_PREFIX + "." + e.getKey() + "=" + e.getValue());
```

Comment:
You need to make this clear in docs that below doesn't work(like it works with shell and sinks):
```
cat ../config/hadoop.properties 
# Use servers.yml to change URI for namenode
# You can add additional properties in this file
spring.hadoop.security.authMethod=kerberos
spring.hadoop.security.namenodePrincipal=hdfs/neo@LOCALDOMAIN
spring.hadoop.security.rmManagerPrincipal=yarn/neo@LOCALDOMAIN
spring.hadoop.security.userKeytab=/usr/local/hadoops/jvalkealahti.keytab
spring.hadoop.security.userPrincipal=jvalkealahti/neo
mapreduce.jobhistory.principal=mapred/neo@LOCALDOMAIN
mapreduce.framework.name=yarn
```
but yml is fine:
```
# Hadoop properties
spring:
  hadoop:
    fsUri: hdfs://localhost:8020
    resourceManagerHost: localhost
    resourceManagerPort: 8032
    security:
      authMethod: kerberos
      namenodePrincipal: hdfs/neo@LOCALDOMAIN
      rmManagerPrincipal: yarn/neo@LOCALDOMAIN
      userKeytab: /usr/local/hadoops/jvalkealahti.keytab
      userPrincipal: jvalkealahti/neo
    config:
      mapreduce.jobhistory.principal: mapred/neo@LOCALDOMAIN
```
in a perfect world same hadoop.properties format should also work.